### DISCUSSION TITLE
Sonoff ZBDongle-P estendere la copertura.

### DISCUSSION THREAD

**Discussion by Robyxxx** [indomus/forum#1962] [GENERALE - Discussioni generiche]
Buongiorno a tutti,
ho configurato correttamente il dongle Zigbee Sonoff con Home Assistant su RPI4 con docker. Fatto il pairing di un paio di sensori a batteria ma purtroppo la copertura è scarsa (in alcuni punti il LQI è pericolosamente vicino a 0, RSSI sembra non disponibile a causa del chip TI CC2652P che non lo supporta). Il mio dongle è il ZBDongle-P (CC2652P + CP2102) ho letto da quache parte che la potenza è imitata a 9dBm e potrebbe essere aumentata a 20 dBm. C'è una procedura per farlo? Oppure in alternativa è possibile aggiungere un extender? Come?

**Comment by snakuzzo** [comment]
Guarda che è una delle chiavette con antenna migliori (se non la migliore) a livello di copertura.
Quell'impostazione la trovi sulla webgui di zigbee2mqtt.
Ricordati che la copertura dipende sempre da 10000000 di fattori...e dalle singole situazioni (se ci sono interferenze, pareti particolari, etc).

**Reply by Robyxxx** [reply]
Io ho Home Assistant CORE su docker, quindi non saprei come installare zigbee2mqtt. C'è un modo per farlo?
Oppure posso programmare la potenza trasmissiva collegando il dongle al PC dove installa provvisoriamente Home Assistant OS su VMWare per poi rispostare tutto su RPI4? Il settaggio viene programmato in flash o una volta scollegata torna il valore di default?
In alternativa esiste un extender? Grazie per le risposte!

**Reply by snakuzzo** [reply]
Installi zigbee2mqtt su docker. Se cerchi trovi la guida sul nostro sito...
Ed è una config a livello di zigbee2mqtt...non a livello di dongle

**Reply by Robyxxx** [reply]
Grazie mille. Proverò con la vostra guida. Se volessi aggiungere router per estendere la copertura hai qualche consiglio?

**Reply by snakuzzo** [reply]
Gli extender sarebbero i "router" (si chiamano così). Solitamente qualsiasi device zigbee alimentato NON a batteria fa anche da router.
Non è detto che tutti funzionino con tutti...potrebbe capitare che alcuni vadano solo su certi canali, altri che danno problemi con device di produttori diversi, etc.
Insomma...devi provare...

Se poi vuoi andare più sul sicuro puoi comprare altre chiavette o board con chip CCxxxx (quindi sempre chip di Texas Instruments) e flashare con firmware di tipo router.
